src/App.scss

```
@@ -1 +1,8 @@
-// styles go here
+html {
+  font-family: Helvetica, sans-serif;
+  font-size: 15px;
+}
+
+body {
```

Let's make box-sizing the same for all elements
```suggestion
* {
```